### DISCUSSION TITLE
fromitive-blog/project/2023-07-09-backtest/

### DISCUSSION THREAD

**Discussion by giscus** [fromitive/fromitive-blog#6] [Q&A]
# fromitive-blog/project/2023-07-09-backtest/

TA-lib를 이용하여 다양한 캔들 차트 전략을 자동으로 적용해보기

https://fromitive.github.io/fromitive-blog/project/2023-07-09-backtest/

**Comment by DDONONG** [comment]
폭발적인 전략 기대합니다
